Tutorials/get-the-tutorials.md

```diff
@@ -10,9 +10,17 @@ SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
 # Getting the tutorials
 ## Python Dependencies
 The CAmkES python dependencies are required to build the [tutorials](ReworkedTutorials). To install you can run:
-```
+```sh
 pip3 install --user camkes-deps
 ```
+
+If you get an error saying the Python 'environment is externally managed' follow the below instructions:
+```sh
+python3 -m venv venv
+./venv/bin/pip install camkes-deps
+source ./venv/bin/activate
```

Comment:
Is that a one time operation, or do you need to do this every time after logging in?

If you need to do that every time to activate a Python environment I'd separate it and say that explicitly. And assuming `venv` is the name of the environment, wouldn't it be clearer to have "seL4" as part of its name?

Reply:
Have addressed that in the additional commit now.